src/main/java/org/fenixedu/spaces/domain/accessControl/SpecificSpaceManagersGroup.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright © 2002 Instituto Superior Técnico
+ *
+ * This file is part of FenixEdu Academic.
+ *
+ * FenixEdu Academic is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * FenixEdu Academic is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with FenixEdu Academic.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.fenixedu.spaces.domain.accessControl;
+
+import static org.fenixedu.bennu.FenixEduSpaceConfiguration.BUNDLE;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.fenixedu.bennu.core.annotation.GroupOperator;
+import org.fenixedu.bennu.core.domain.User;
+import org.fenixedu.bennu.core.groups.DynamicGroup;
+import org.fenixedu.bennu.core.groups.GroupStrategy;
+import org.fenixedu.bennu.core.i18n.BundleUtil;
+import org.fenixedu.spaces.domain.Space;
+import org.joda.time.DateTime;
+
+@GroupOperator("specificSpaceManagersGroup")
```

Comment:
it is called 'specific' to follow the naming convention of existing groups that mimic this behavior and usage